README.md

```diff
@@ -1,17 +1,22 @@
 # Amio
 
-Repository for reporting issues and feature requests of [Amio Project](http://amio.io/).
+Main Amio repository. You can find Amio SDKs, links to documentation, report issues or feature requests.
```

Comment:
You can find Amio SDKs and links to documentation here. Feel free to report issues or submit feature requests.

co takhle?